### PR TITLE
Design System - Document ways of consuming the DS

### PR DESCRIPTION
**Pull request by boxadesign**
### What is the context of this PR?
Created a new set of pages under /get-started

### CODE REVIEW

**Review by rmccar — APPROVED** [review]

**Comment by jrbarnes9** [comment]
Think we need something on the top level "Get started" page to introduce the installation pages
<details><summary>Browser metadata</summary>





```
Path:      /get-started/
Browser:   Chrome 91.0.4472.106 on Mac OS 10.14.6
Viewport:  1280 x 698 @2x
Language:  en
Cookies:   Enabled
```

[Open in BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-1576--ons-design-system.netlify.app%2Fget-started%2F&os=OS+X&os_version=Mojave&browser=Chrome&browser_version=91.0&resolution=1280x698&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-1576--ons-design-system.netlify.app/get-started/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60e5a9c336b60b0008207b6a&commentID=60e7fe2770d6da85b59f951a&resolution=resolved)

**Comment by jrbarnes9** [comment]
Maybe we shouldn't spell out CSS in the heading, because it's not great for a nav item when the devs will be looking for "CSS"
screenshot


<details><summary>Browser metadata</summary>





```
Path:      /get-started/css-js-assets/
Browser:   Chrome 91.0.4472.106 on Mac OS 10.14.6
Viewport:  1280 x 698 @2x
Language:  en
Cookies:   Enabled
```

[Open in BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-1576--ons-design-system.netlify.app%2Fget-started%2Fcss-js-assets%2F&os=OS+X&os_version=Mojave&browser=Chrome&browser_version=91.0&resolution=1280x698&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-1576--ons-design-system.netlify.app/get-started/css-js-assets/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60e5a9c336b60b0008207b6a&commentID=60e7fecd922616820a1aeab7&resolution=resolved)

**Comment by jrbarnes9** [comment]
Don't think it makes sense to spell it out here either.  There's also a broken link in the last bullet point.
screenshot


<details><summary>Browser metadata</summary>





```
Path:      /get-started/npm/
Browser:   Chrome 91.0.4472.106 on Mac OS 10.14.6
Viewport:  1280 x 698 @2x
Language:  en
Cookies:   Enabled
```

[Open in BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-1576--ons-design-system.netlify.app%2Fget-started%2Fnpm%2F&os=OS+X&os_version=Mojave&browser=Chrome&browser_version=91.0&resolution=1280x698&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-1576--ons-design-system.netlify.app/get-started/npm/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60e5a9c336b60b0008207b6a&commentID=60e7ffd492261684731aeb03&resolution=resolved)

File: src/get-started/css-js-assets/index.njk [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+@import 'node_modules/@ons/design-system/scss/helpers/index';
+@import 'node_modules/@ons/design-system/scss/utilities/index';
+@import 'node_modules/@ons/design-system/scss/base/index';
+@import 'node_modules/@ons/design-system/scss/objects/index';" }) }}
+<br>
+Importing these files ensures components and patterns have access to any required global classes and utilities.
+
+To import the styles for a single component, for example, the button component, you need to add this after the base Sass files:
+
+{{ onsCodeHighlight({ "code": "@import 'node_modules/@ons/design-system/components/button/button';" }) }}
```
Comment (jrbarnes9) [inline src/get-started/css-js-assets/index.njk:42]: Just checking `/button/button` is correct and not a typo?

**Comment by jrbarnes9** [comment]
I don't think Markdown likes tags written in the `` markdown so probably need to use `<code>`
screenshot


<details><summary>Browser metadata</summary>





```
Path:      /get-started/css-js-assets/#cdn
Browser:   Chrome 91.0.4472.106 on Mac OS 10.14.6
Viewport:  1280 x 698 @2x
Language:  en
Cookies:   Enabled
```

[Open in BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-1576--ons-design-system.netlify.app%2Fget-started%2Fcss-js-assets%2F%23cdn&os=OS+X&os_version=Mojave&browser=Chrome&browser_version=91.0&resolution=1280x698&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-1576--ons-design-system.netlify.app/get-started/css-js-assets/#cdn) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60e5a9c336b60b0008207b6a&commentID=60e803b8aa49478357674dd0&resolution=resolved)

**Comment by boxadesign** [comment]
> **A reviewer left a comment:**
> 
> Exporting SVGs and Browser compatibility pages need updating to new content in Google docs
> 
> Browser metadata
> [Open Deploy Preview](https://deploy-preview-1576--ons-design-system.netlify.app/get-started/browser-compatibility/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60e5a9c336b60b0008207b6a&commentID=60e80cb780ba80a27a0b3596&resolution=resolved)

This won't be done in this PR.

**Comment by boxadesign** [comment]
> **A reviewer left a comment:**
> 
> Design assets:
> Can you change the first two paras so they match what's in the Google doc:
> We use Sketch to mock up designs and create click-through prototypes to demonstrate user journeys and carry out user testing.
> First, download Sketch to your device. You then need the ONS Sketch Library to create designs and prototypes that are consistent with our brand and styles.
> 
> Browser metadata
> [Open Deploy Preview](https://deploy-preview-1576--ons-design-system.netlify.app/get-started/design-assets/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60e5a9c336b60b0008207b6a&commentID=60e80c38423fcc920cea64d4&resolution=resolved)

This won't be done in this pr

**Comment by jrbarnes9** [comment]
`<head>` tag still not rendering 
screenshot


<details><summary>Browser metadata</summary>





```
Path:      /get-started/css-js-assets/
Browser:   Chrome 91.0.4472.106 on Mac OS 10.14.6
Viewport:  1920 x 978 @1x
Language:  en
Cookies:   Enabled
```

[Open in BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-1576--ons-design-system.netlify.app%2Fget-started%2Fcss-js-assets%2F&os=OS+X&os_version=Mojave&browser=Chrome&browser_version=91.0&resolution=1920x978&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-1576--ons-design-system.netlify.app/get-started/css-js-assets/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60eee963015a120007e510bb&commentID=60ef051c162c135c45702e5b&resolution=resolved)

File: src/get-started/npm/index.njk [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    })
+}}. Every release is published to npm and provides a quick way to install everything you need. 
+Inside the package are the following:
+
+- **components** – each component contains a macro file that can be used in a Nunjucks-compatible application
+- **Sass files** – all Sass rules are available to import, either everything or specific parts like a single component or the Sass that controls layout
+- **CSS files** – Compiled CSS of all components and layout styles
+- **JS files** – bundled JavaScript modules
+- **fonts, images, favicons** – static assets used in components and templates
+- **base [page template](styles/page-template)** – a Nunjucks template that contains the core markup and components needed to create an ONS service.
```
Comment (jrbarnes9) [inline src/get-started/npm/index.njk:23]:
```suggestion
- **base [page template](/styles/page-template)** – a Nunjucks template that contains the core markup and components needed to create an ONS service.
```

**Review by jrbarnes9 — DISMISSED** [review]

**Review by LauraC119 — COMMENTED** [review]
Using the CSS, JS and assets page:
1. Change the note you've added to:
Note that some local development environments require you to override the the global ONS_assets_base_URL variable to use the chunked JS files. To do this, add the following script before the reference to the main.js file:
2. Change the two "you're"s to "you are"

Install NPM page:
1. change "Compiled" on the bullet list to lower case
2. remove full stop on the end of the bullet list

**Review by jrbarnes9 — APPROVED** [review]

**Review by LauraC119 — CHANGES_REQUESTED** [review]
So sorry, I made a mistake in that new paragraph. Can you remove the duplicated "the":
"require you to override the **the** global ONS_assets_base_URL variable"

**Review by LauraC119 — APPROVED** [review]

**Review by jrbarnes9 — APPROVED** [review]

**Pull request merged**